conn.go

```diff
@@ -1356,28 +1356,35 @@ func (c *Conn) query(ctx context.Context, statement string, values ...interface{
 
 func (c *Conn) awaitSchemaAgreement(ctx context.Context) (err error) {
 	const (
-		peerSchemas  = "SELECT schema_version, peer FROM system.peers"
+		peerSchemas  = "SELECT * FROM system.peers"
 		localSchemas = "SELECT schema_version FROM system.local WHERE key='local'"
 	)
 
 	var versions map[string]struct{}
+	var schemaVersion string
```

Comment:
this isnt used?

Reply:
a bit further in the loop, [L1395](https://github.com/gocql/gocql/pull/1300/files#diff-a2ddca1600f34abb91244424f8431c2cR1395) and on